sourcepolicy/engine.go

```diff
@@ -74,15 +74,13 @@ func (e *Engine) Evaluate(ctx context.Context, op *pb.Op) (bool, error) {
 			return mutated, errors.Wrapf(ErrTooManyOps, "too many mutations on a single source")
 		}
 
-		srcOp := op.GetSource()
-		if srcOp == nil {
```

Comment:
I noticed we are passing `op.GetSoruce()` directly into `Evaluate`.
Do we need a nil check?

Reply:
There is a nil check in line 65